portworx.tf

```diff
@@ -47,12 +47,11 @@ resource "kubernetes_secret" "etcd" {
 ##################################
 resource "ibm_resource_instance" "portworx" {
   depends_on = [
-    ibm_container_storage_attachment.volume_attach,
```

Comment:
do we need to separate the modules to support cloud drive vs non cloud drive as this variable would be required for installation without cloud drive?